config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -330,12 +332,17 @@ teams:
   cluster-api-provider-vsphere-maintainers:
     description: Write access to the cluster-api-provider-vsphere repo
     members:
+    - akutz
     - chrischdi
+    - figo
+    - frapposelli
     - gab-satchi
     - killianmuldoon
     - randomvariable
     - sbueringer
+    - sidharthsurana
     - srm09
+    - vincepri
```

Comment:
These can also be dropped. Ref: https://github.com/kubernetes/org/pull/4345

Reply:
Note: Changes to the whole file can be dropped. GitHub UI is weird and doesn't show the whole selection.

Reply:
dropped it as part of the respective entire commit. Thanks!

Reply:
Actually, just realised. I missed this. Will raise a PR to fix this tomorrow,